### PR TITLE
BR-72: Add translations

### PR DESCRIPTION
**Pull request by jeissonmoncada001**
## Jira link

[BR-72](https://mundobakia.atlassian.net/browse/BR-72)

## Description

Add translations

## How do I test this PR?

- Clone the repository
- Install the dependencies using the 'yarn' command
- Run the application locally using 'yarn start' command

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,13 @@
     "@types/node": "^17.0.23",
     "@types/react": "^17.0.20",
     "@types/react-dom": "^17.0.9",
+    "i18next": "^21.6.14",
+    "i18next-browser-languagedetector": "^6.1.4",
+    "i18next-http-backend": "^1.4.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-i18next": "^11.16.2",
+    "react-router-dom": "^6.3.0",
```
Comment (Julian9411) [inline package.json:23]: Why add react-router-dom

File: src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,13 @@
-import React, { FC } from 'react';
+import React, { FC, Suspense } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
 import Home from './common/Home';
 
-const App: FC = () => <Home />;
+const App: FC = () => (
+  <Suspense fallback={null}>
```
Comment (Julian9411) [inline src/App.tsx:6]: why this Suspense?

**Review by Julian9411 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jeissonmoncada001) [reply package.json]: Done!

File: src/App.tsx (hunk 2 above)
Comment (jeissonmoncada001) [reply src/App.tsx]: Deleted as not necessary

**Review by Julian9411 — APPROVED** [review]

**Pull request merged**